docs/_news/news10.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: news
+title: New for Codewind 0.10.0
+description: New for Codewind 0.10.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news10
+---
+
+## Codewind 0.10.0
+Thursday, 19 March 2020
+
+#### ✨ New Features and Highlights for 0.10.0 ✨
+**IntelliJ Tech Preview Updates**
+
+As promised, we've continue to work hard on getting the IntelliJ experience up and consistent with our other IDEs. Here are the latest IntelliJ updates:
+
+- [View project log files in IntelliJ](https://github.com/eclipse/codewind/issues/989).
+- Open and [view the performance dashboard](https://github.com/eclipse/codewind/issues/2038) for select project types.
+- [Enable and disable projects](https://github.com/eclipse/codewind/issues/987).
+- You can [simulatneously select and remove multiple projects](https://github.com/eclipse/codewind/issues/2225).
+- Create a new project with the [`New Project` menu action](https://github.com/eclipse/codewind-intellij/pull/64)
+![](images/imagesfornews/newprojintellij.gif){:width="800"}
+
+**IDEs (Both Eclipse and VS Code)**
+- The [Performance Dashboard will always be enabled](https://github.com/eclipse/codewind/issues/2299), even if the project does not have metrics available. 
+    - Note that for [projects without metrics, running loads will fail](https://github.com/eclipse/codewind/issues/2384)
+
+**VS Code**
+- You can use the profiling extension for [remote connections](https://github.com/eclipse/codewind/pull/2123) and in [Che](https://github.com/eclipse/codewind-vscode/pull/457).
+- [The “add to workspace” Project command has been reworked for multi-root workspaces](https://github.com/eclipse/codewind-vscode/pull/484). It will no longer appear on projects that is already a root workspace folder, and will give an overrideable warning if run on a project that is in a subfolder of the workspace.
```

Comment:
- Maybe put `add to workspace` in code phrase format instead of quotation marks.
- I don't think "Project" needs capitalization.
- "is" instead of "has been"
- "It no longer appears on projects that are already root workspace folders..." instead of "It will no longer appear on projects that is..."
- No comma after "folder"
- "gives" instead of "will give"